service.yml

```diff
@@ -12,6 +12,7 @@ codeowners:
 semaphore:
   enable: true
   pipeline_enable: false
+  status_level: block
```

Comment:
Setting this with the hope that follow-on branches will show each block in the pipeline as separate "checks"